charts/zabbix/artifacthub-pkg.yml

```diff
@@ -5,7 +5,7 @@
 # https://github.com/kedacore/external-scalers/blob/main/artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml
 # https://artifacthub.io/packages/keda-scaler/keda-official-external-scalers/external-scaler-azure-cosmos-db?modal=install
 
-version: 6.0.0 # helm chart version
+version: 6.1.0 # helm chart version
```

Comment:
Hi @sinux-l5d!

Can you remove this version? I will choose the correct number during the publish process.